clang/lib/CConv/ConstraintVariables.cpp

```diff
@@ -44,9 +44,10 @@ bool ConstraintVariable::isChecked(const EnvironmentMap &E) const {
 
 PointerVariableConstraint *
 PointerVariableConstraint::getWildPVConstraint(Constraints &CS,
-                                               const std::string &Rsn) {
-  VarAtom *VA = CS.getFreshVar("wildvar", VarAtom::V_Other);
-  CS.addConstraint(CS.createGeq(VA, CS.getWild(), Rsn, true));
+                                               const std::string &Rsn,
+                                               PersistentSourceLoc *PSL) {
+  VarAtom *VA = CS.getVarGEQ("wildvar", VarAtom::V_Other,
```

Comment:
I don't care that much, but `getVar...` is not a very intuitive name for me. Before it was clear you were creating the constraint. I think you still are, right? Maybe `createFreshGEQ` or something like that?